h2o-docs/src/product/training-models.rst

```diff
@@ -428,7 +428,7 @@ This example uses the :ref:`isoforest` algorithm to detect anomalies in the Elec
 Dimensionality Reduction Example
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Below is a simple example showing how to use the :ref:`GLRM <glrm>` algorithm for dimensionality reduction of the USArrests dataset.
+This code demonstrates how to apply the :ref:`GLRM <glrm>` algorithm for dimensionality reduction on the USArrests dataset using H2O-3. It includes examples in both R and Python for importing data, splitting it into training and validation sets, training the GLRM model, evaluating its performance, and generating predictions. The purpose of this example is to show how GLRM can reduce data dimensionality while preserving essential information, making it useful for feature extraction and data preprocessing.
```

Comment:
please replace

generating predictions

to

reconstructing the dataset when predict is called.